### DISCUSSION TITLE
numba depreciation warning.

### DISCUSSION THREAD

**Discussion by chabazite** [shap/shap#3016] [Q&A] [answered]
Recently, with V 0.41.0, I have been receiving numba depreciation warning. Is there a plan to update to be compatible with this new version of numba, does this have an impact on my results, is there a way for me to fix this warning on my end?

See the warning below:

`NumbaDeprecationWarning: The 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.`

**Comment by thatlittleboy** [comment, marked as answer]
This issue was addressed by https://github.com/slundberg/shap/pull/2943. Please use the master branch in the meantime. A PyPI release is coming soon.
